actions/post-preview-urls/dist/main/index.js

```diff
@@ -24305,7 +24307,7 @@ async function postPreviewUrls({
     }
     return `_${link.name}_: ${link.url}`;
   }).join(asLabels ? ", " : "\n");
-  const heading = `**${appName} preview links**`;
+  const heading = `**${[appIcon, appName].join(" ").trim()} preview links**`;
```

Comment:
This seems like the same string as in `actions/post-preview-urls/index.ts` -- would it make sense to share it? In case it changes in the future?

Reply:
This is the build artefact of that file, it needs to be committed to git

Reply:
Yeah, I just meant to extract out a util of sorts to share across the files... smth like

```typescript
const generateHeading = (appIcon: string, appName: string): string => {
  return `**${[appIcon, appName].join(" ").trim()} preview links**`;
};```

In case it changes down the line, so we change both at the same time. 

Might be totally overkill, then just ignore 👍🏻

Reply:
Not sure I understand, this file is produced by running `make build` in the `actions/post-preview-urls` folder. 
`actions/post-preview-urls/index.ts` is the input file and `actions/post-preview-urls/dist/main/index.js` is the output file.

Reply:
🙃 ah, gotcha! -- my bad, monday brain. I wouldn't have thought we'd commit those files tbh

Reply:
Yeah, it needs to be committed due to the way reusable github actions are set up (product-web fetches the action directly from the git repo)